Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -691,6 +696,7 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				COPY_PHASE_STRIP = NO;
+				DEAD_CODE_STRIPPING = YES;
```

Comment:
Do you know what the purpose of this setting is? and why it needs to be set to yes?